src/lib/server/websearch/generateQuery.ts

```diff
@@ -69,7 +69,7 @@ Current Question: Where is it being hosted ?`,
 							.map(({ content }) => `- ${content}`)
 							.join("\n")}`
 					: "") +
-				"\n\nCurrent Question:" +
+				"\n\nCurrent Question: " +
```

Comment:
we need space here. Otherwise it was becoming something like `Current Question:who is Elon Musk`